dGame/dComponents/CharacterComponent.h

```diff
@@ -286,6 +288,11 @@ class CharacterComponent : public Component {
      */
 	uint32_t m_Level;
 
+    /**
+     * Maximum level of the entity
+     */
+    uint64_t m_MaxLevel;
```

Comment:
Quite sure this is a `uint32_t`

Reply:
UScore is stored as `unit64_t` so I can understand the mistake here

Reply:
Or I could have looked up a couple of lines to see that level is stored as uint32_t lol